honeycomb-registry-client/index.js

```diff
@@ -26,7 +28,7 @@ function register(name, host, port) {
 
   const eureka = {
     host: process.env.EUREKA_HOST || 'localhost',
-    port: process.env.EUREKA_PORT || '1111',
+    port: process.env.EUREKA_PORT || '8761',
```

Comment:
The default port in each tutorial, see https://github.com/spring-cloud-samples/eureka